install/connections.csv

```diff
@@ -617,32 +617,32 @@ OT001,04,Waimangu Inferno Crater,,,2019-10-11T00:50:00Z,9999-01-01T00:00:00Z
 OT013,01,Rotoma Soda Spring,,,2021-03-17T22:20:00Z,9999-01-01T00:00:00Z
 OT013,02,Rotoma Soda Spring,,,2021-03-17T22:20:00Z,9999-01-01T00:00:00Z
 OT013,03,Rotoma Soda Spring,,,2022-06-28T23:00:00Z,9999-01-01T00:00:00Z
-OTAT,40,Dunedin Tsunami,,,2010-02-25T00:00:00Z,9999-01-01T00:00:00Z
-OTAT,41,Dunedin Tsunami,,,2010-02-25T00:00:00Z,9999-01-01T00:00:00Z
+OTAT,40,Dunedin Tsunami,,0,2010-02-25T00:00:00Z,9999-01-01T00:00:00Z
+OTAT,41,Dunedin Tsunami,,1,2010-02-25T00:00:00Z,9999-01-01T00:00:00Z
 OTKO,10,Otekaieke,Portable,,2014-09-18T01:00:00Z,2015-04-11T05:00:00Z
-OTVZ,10,Oturere,,,2004-11-18T04:00:00Z,9999-01-01T00:00:00Z
-OTVZ,30,Oturere,,,2013-10-02T00:00:00Z,9999-01-01T00:00:00Z
-OUZ,10,Omahuta,,,2002-08-30T08:00:00Z,9999-01-01T00:00:00Z
+OTVZ,10,Oturere,,0,2004-11-18T04:00:00Z,9999-01-01T00:00:00Z
+OTVZ,30,Oturere,,3,2013-10-02T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
how would we handle "numbers" for a site that have
- weak motion (0-2)
- strong motion (???)
- acoustic (3)

Reply:
they all can't connect to a single digitser (at the moment we only have a maximum of 6 channel mdels). i.e. weak + strong, or weak + something else, and so far the "else" has been unique.